packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -88,11 +88,20 @@ const CARDANO_FEE_INFO: FeeInfoWithLevels = {
     dustLimit: -1, // unknown/unused
 };
 
+const SOLANA_FEE_INFO: FeeInfoWithLevels = {
+    blockTime: -1, // unknown
+    defaultFees: [{ label: 'normal', feePerUnit: '5000', blocks: -1 }],
+    minFee: 5000,
+    maxFee: 5000,
+    dustLimit: -1, // unknown/unused
+};
```

Comment:
I have zero knowledge about fees in solana. this means there is only static fee? always 5000?

Reply:
Hm, it's actually 5000*`number_of_signatures`. Solana tx size is limited to 1232B, one signature + public key is 96B so the limit is 5000*(1232/96) -> 12. I'm not sure how this value is used in Suite so I'm not sure if we should change it to 5000*12. @memoriesadrift could you please take a look? 🙏

Reply:
Please change to 5000 * 12 in that case to be accurate to the spec.

The rationale raised for this when it was added was that the functionality in suite only ever needs 1 signature, i.e. the default fee is always 5000. 
Fees for token account funding etc. are not accounted for in the max either, since they can essentially be arbitrarily high, is that okay?

Reply:
> Fees for token account funding etc. are not accounted for in the max either, since they can essentially be arbitrarily high, is that okay?

I think those shouldn't be counted as fees.

`5000*12` added in https://github.com/trezor/trezor-suite/pull/9958/commits/8e5bb47a6550e723d4fca78b9d7923111d1b679e.

Reply:
As far as I know, both `minFee` and `maxFee` represent values per unit (same as `feePerUnit`) and both of them are usually used per (v)byte. So I want to ask, do you plan to make changes in Suite so the default unit for Solana is signature rather than byte, and if you do, shouldn't the `maxFee` be same as the `minFee`? If not, it may be necessary to adjust the values to bytes.

Reply:
The `minFee` and `maxFee` values are actually not used for Solana as the fee in Suite is constant - there is always only 1 signature per transaction and so the fee is always 5000 Lamports. `feePerUnit` is basically used as `feePerSignature` in `sendFormSolanaActions.ts` ([here](https://github.com/vacuumlabs/trezor-suite/blob/af8edc0616a010b515fb2a59c5882a6dc41fe4d3/packages/suite/src/actions/wallet/send/sendFormSolanaActions.ts#L35)). I changed the values to `-1` in https://github.com/trezor/trezor-suite/pull/9958/commits/a532ce4e2dc77e703779780ac4cd80cc8b59923e and added a comment in https://github.com/trezor/trezor-suite/pull/9958/commits/7778837ab8c6a82d2f3f159441372be643ab42db.

Reply:
Thanks for the info, I'll look at it again in UI PR.